### PR TITLE
add docs for maya ornatrix

### PR DESCRIPTION
**Pull request by moonyuet**
Add documentation for the Ornatrix implemented developed with `ayon-maya` PR: https://github.com/ynput/ayon-maya/pull/1

### CODE REVIEW

File: website/docs/addon_maya_ornatrix_artist.md [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+
+Once you finished on your Ornatrix rig, you can also go to `AYON` -> `Publish`. It will automatically create Ornatrix Cache instance and allow you to publish Ornatrix Alembic.
+You can turn off the auto-creation by going to `AYON` server -> Project settings > `Maya` > `Loaders` > `Ornatrix Rig Loader` -> `Create Ornatrix Cache on load`
+
+
+## Ornatrix Caches
+
+### Creating and publishing
+
+Users can either create ```Ornatrix Cache``` instance by their own or load ornatrix rig which automatically create the instance after launching the Publisher UI.
```
Comment (BigRoy) [inline website/docs/addon_maya_ornatrix_artist.md:69]:
```suggestion
Users can either create ```Ornatrix Cache``` instance by their own or load ornatrix rig which automatically creates the instance after launching the Publisher UI.
```

File: website/docs/addon_maya_ornatrix_artist.md [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+
+The cache family would export Ornatrix Alembic(.abc) which includes the Ornatrix data into the publish folder.
```
Comment (BigRoy) [inline website/docs/addon_maya_ornatrix_artist.md:71]:
```suggestion
The cache family exports Ornatrix Alembic (.abc) which includes the Ornatrix data into the publish folder.
```

File: website/docs/addon_maya_ornatrix_artist.md [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+
+![Maya - Ornatrix Loader Product Type](assets/maya/artist/ox_loader_product_type.png)
+
+```Ornatrix Rig```supports to load maya scene (.ma), groom preset (.oxg.zip), Ornatrix alembic (.abc) and Ornatrix alembic to Hair Guide Node (.abc).
+
+![Maya - Ornatrix Loaders](assets/maya/artist/ox_loader_product_type.png)
+
+Both Maya Scene and groom preset allow for connecting ornatrix rig with the animation/pointcache rig.
+
+Users need to go to `AYON` -> `Manage...`and select the assets from oxrig and animation/pointcache product type. Then they need to right-click `Actions` -> `Connect Ornatrix Rig` and the Ornatrix geometry would be connected the animation rig with the transform and mesh attributes.
```
[the remark this answers is not in the document]
Comment (moonyuet) [reply website/docs/addon_maya_ornatrix_artist.md:57]: yes, I will make a gif to explain that

File: website/docs/addon_maya_ornatrix_artist.md [hunk 4]
```diff
@@ -0,0 +1,74 @@
...
+---
+
+import ReactMarkdown from "react-markdown";
+import versions from '@site/docs/assets/json/Ayon_addons_version.json'
+
+<ReactMarkdown>
+{versions.Maya_Badge}
+</ReactMarkdown>
+
+AYON supports Ephere's **Ornatrix** with the workflow of publishing and loading maya scene with Ornatrix data, Ornatrix groom presets and Ornatrix Alembic.It also supports to connect your target geometry with the loaded Ornatrix data(either from the maya scene or the groom preset)
```
Comment (BigRoy) [inline website/docs/addon_maya_ornatrix_artist.md:14]:
```suggestion
AYON supports Ephere's **Ornatrix** with the workflow of publishing and loading maya scene with Ornatrix data, Ornatrix groom presets and Ornatrix Alembic. It also supports to connect your target geometry with the loaded Ornatrix data (either from the maya scene or the groom preset.)
```

File: website/docs/addon_maya_ornatrix_artist.md [hunk 5]
```diff
@@ -0,0 +1,74 @@
...
+![Maya - Ornatrix Loader Setting](assets/maya/artist/ox_loaders_maya_addon_setting.png)
+
+## Workflow
+AYON has two creators for Ornatrix publishing: ```Ornatrix Rig``` and ```Ornatrix Cache```.
+
+## Ornatrix Rig
+
+### Creating and publishing
+
+```Ornatrix Rig```(product type: *oxrig*) being to publish the asset with the Ornatrix data. Users select your mesh with Ornatrix fur and create ```Ornatrix Rig``` instance. When it is published, several files are exported to the publish folder:
```
Comment (BigRoy) [inline website/docs/addon_maya_ornatrix_artist.md:39]:
```suggestion
The **Ornatrix Rig** (product type: *oxrig*) publishes the geometry with the Ornatrix data. 

Select your mesh (that has with Ornatrix fur) and create the Ornatrix Rig instance. When it is published, several files are exported to the publish folder:
```

File: website/docs/addon_maya_ornatrix_artist.md [hunk 6]
```diff
@@ -0,0 +1,74 @@
...
+- Groom Preset (.oxg.zip)
+- Rig Setting data (.rigsettings)
+- Ornatrix Almebic (.abc)
+- Cache Setting data (.cachesettings)
+
+### Loading
+
+![Maya - Ornatrix Loader Product Type](assets/maya/artist/ox_loader_product_type.png)
+
+```Ornatrix Rig```supports to load maya scene (.ma), groom preset (.oxg.zip), Ornatrix alembic (.abc) and Ornatrix alembic to Hair Guide Node (.abc).
```
Comment (BigRoy) [inline website/docs/addon_maya_ornatrix_artist.md:51]:
```suggestion
**Ornatrix Rig** supports to load maya scene (.ma), groom preset (.oxg.zip), Ornatrix alembic (.abc) and Ornatrix alembic to Hair Guide Node (.abc).
```

File: website/docs/addon_maya_ornatrix_artist.md [hunk 7]
```diff
@@ -0,0 +1,74 @@
...
+![Maya - Ornatrix Loaders](assets/maya/artist/ox_loader_product_type.png)
+
+Both Maya Scene and groom preset allow for connecting ornatrix rig with the animation/pointcache rig.
+
+Users need to go to `AYON` -> `Manage...`and select the assets from oxrig and animation/pointcache product type. Then they need to right-click `Actions` -> `Connect Ornatrix Rig` and the Ornatrix geometry would be connected the animation rig with the transform and mesh attributes.
+
+Users can choose to load Hair Guide Node with Alembic by Ornatrix alembic to Hair Guide Node (.abc), which sets the alembic as file cache path inside the node and they can select to connect the node with the node editor or edit it in the operator stack editor.
+
+Once you finished on your Ornatrix rig, you can also go to `AYON` -> `Publish`. It will automatically create Ornatrix Cache instance and allow you to publish Ornatrix Alembic.
+You can turn off the auto-creation by going to `AYON` server -> Project settings > `Maya` > `Loaders` > `Ornatrix Rig Loader` -> `Create Ornatrix Cache on load`
```
Comment (BigRoy) [inline website/docs/addon_maya_ornatrix_artist.md:62]:
```suggestion
Once you are finished with the Ornatrix rig, you can go to `AYON` -> `Publish` to publish an Ornatrix Cache. 

:::tip
Loading the Ornatrix Rig will automatically create the Ornatrix Cache instance for it. That's why you can directly continue publishing data when you're ready.

You can turn off the auto-creation by going to `AYON` server -> Project settings > `Maya` > `Loaders` > `Ornatrix Rig Loader` -> `Create Ornatrix Cache on load`
:::
```

File: website/docs/addon_maya_ornatrix_artist.md [hunk 8]
```diff
@@ -0,0 +1,74 @@
...
+
+
+## Ornatrix Caches
+
+### Creating and publishing
+
+Users can either create ```Ornatrix Cache``` instance by their own or load ornatrix rig which automatically creates the instance after launching the Publisher UI.
```
Comment (BigRoy) [inline website/docs/addon_maya_ornatrix_artist.md:69]:
```suggestion
Users can either create an ```Ornatrix Cache``` instance themselves or load ornatrix rig which automatically creates the instance ready for publishing.
```

File: website/docs/addon_maya_ornatrix_artist.md [hunk 9]
```diff
@@ -0,0 +1,74 @@
...
+
+The cache family exports Ornatrix Alembic (.abc) which includes the Ornatrix data into the publish folder.
+
+### Loading
+```Ornatrix Rig```supports to load Ornatrix alembic (.abc) and Ornatrix alembic to Hair Guide Node (.abc).
```
Comment (BigRoy) [inline website/docs/addon_maya_ornatrix_artist.md:74]: I'm pretty sure this is not about the Ornatrix Rig but Ornatrix Cache?
```suggestion
The **Ornatrix Cache** supports to load Ornatrix alembic (.abc) and Ornatrix alembic to Hair Guide Node (.abc).
```

File: website/docs/addon_maya_ornatrix_artist.md [hunk 10]
```diff
@@ -0,0 +1,81 @@
...
+- Ornatrix Almebic (.abc)
+- Cache Setting data (.cachesettings)
+
+### Loading
+
+![Maya - Ornatrix Loader Product Type](assets/maya/artist/ox_loader_product_type.png)
+
+**Ornatrix Rig** supports to load maya scene (.ma), groom preset (.oxg.zip), Ornatrix alembic (.abc) and Ornatrix alembic to Hair Guide Node (.abc).
+
+![Maya - Ornatrix Loaders](assets/maya/artist/ox_loader_product_type.png)
```
Comment (BigRoy) [inline website/docs/addon_maya_ornatrix_artist.md:55]: Did you mean another link? It's showing the same image twice.
```suggestion
![Maya - Ornatrix Loaders](assets/maya/artist/ox_rig_loader.png)
```

File: website/docs/addon_maya_ornatrix_artist.md [hunk 11]
```diff
@@ -0,0 +1,81 @@
...
+
+Both Maya Scene and groom preset allow for connecting ornatrix rig with the animation/pointcache rig.
+
+Users need to go to `AYON` -> `Manage...`and select the assets from oxrig and animation/pointcache product type. Then they need to right-click `Actions` -> `Connect Ornatrix Rig` and the Ornatrix geometry would be connected the animation rig with the transform and mesh attributes.
+
+Users can choose to load Hair Guide Node with Alembic by Ornatrix alembic to Hair Guide Node (.abc), which sets the alembic as file cache path inside the node and they can select to connect the node with the node editor or edit it in the operator stack editor.
```
Comment (BigRoy) [inline website/docs/addon_maya_ornatrix_artist.md:61]:
```suggestion
#### Loading Ornatrix Hair Guide using the Ornatrix alembic

Users can choose to load Hair Guide Node with Alembic by Ornatrix alembic to Hair Guide Node (.abc), which sets the alembic as file cache path inside the node and they can select to connect the node with the node editor or edit it in the operator stack editor.
```

File: website/docs/addon_maya_ornatrix_artist.md [hunk 12]
```diff
@@ -0,0 +1,101 @@
...
+
+## Ornatrix Rig
+
+### Creating and publishing
+
+The **Ornatrix Rig** (product type: *oxrig*) publishes the geometry with the Ornatrix data.
+
+![Maya - Ornatrix Rig Publishing](assets/maya/artist/ornatrix_rig_publishing.gif)
+
+Select your mesh (that has with Ornatrix fur) and create the Ornatrix Rig instance.If the instance contains anything other than the mesh, the validator checks and raise error to make sure there are always mesh(s) inside the instance. When it is published, several files are exported to the publish folder:
```
Comment (BigRoy) [inline website/docs/addon_maya_ornatrix_artist.md:43]:
```suggestion
Select your mesh (that has Ornatrix fur) and create the Ornatrix Rig instance.

:::tip
If the instance contains anything other than the mesh, the publish validators will report the error to make sure there are always mesh(es) inside the instance.
:::

When it is published, several files are exported to the publish folder:
```

File: website/docs/addon_maya_ornatrix_artist.md [hunk 13]
```diff
@@ -0,0 +1,101 @@
...
+
+![Maya - Ornatrix Loader Product Type](assets/maya/artist/ox_loader_product_type.png)
+
+**Ornatrix Rig** supports to load maya scene (.ma), groom preset (.oxg.zip), Ornatrix alembic (.abc) and Ornatrix alembic to Hair Guide Node (.abc).
+
+![Maya - Ornatrix Loaders](assets/maya/artist/ox_rig_loader.png)
+
+Both Maya Scene and groom preset allow for connecting ornatrix rig with the animation/pointcache rig.
+
+![Maya - Ornatrix Rig Loading](assets/maya/artist/ornatrix_rig_loading.gif)
```
Comment (BigRoy) [inline website/docs/addon_maya_ornatrix_artist.md:61]: Could it be @moonyuet that these gifs do not loop but stop after first play?

File: website/docs/addon_maya_ornatrix_artist.md [hunk 14]
```diff
@@ -0,0 +1,101 @@
...
+After loading the Ornatrix Rig and a pointcache to connect it to, you can use the AYON tools to connect it for you:
+
+1. Go to `AYON` -> `Manage...`
+2. In the inventory select the assets from oxrig and animation/pointcache product type.
+3. Right-click `Actions` -> `Connect Ornatrix Rig`
+The Ornatrix geometry will be connected the animation rig with the transform and mesh attributes.
+
+:::tip
+The connecting will occur by node name - so if it fails, make sure the node name for the geometry is the same across your animation caches and the ornatrix exports.
+**Double geometry:** Be aware that both the ornatrix rig and your animated cache have the mesh geometry. After connecting the fur you will most likely want to hide one of the two to ensure you do not have two overlapping meshes during rendering.
```
Comment (BigRoy) [inline website/docs/addon_maya_ornatrix_artist.md:73]: Fix markdown formatting to ensure there's a decent line break between these.
```suggestion

**Double geometry:** Be aware that both the ornatrix rig and your animated cache have the mesh geometry. After connecting the fur you will most likely want to hide one of the two to ensure you do not have two overlapping meshes during rendering.
```

File: website/docs/addon_maya_ornatrix_artist.md [hunk 15]
```diff
@@ -0,0 +1,101 @@
...
+
+You can turn off the auto-creation by going to `AYON` server -> Project settings > `Maya` > `Loaders` > `Ornatrix Rig Loader` -> `Create Ornatrix Cache on load`
+:::
+
+
+## Ornatrix Caches
+
+### Creating and publishing
+
+Users can either create an ```Ornatrix Cache``` instance themselves or load ornatrix rig which automatically creates the instance ready for publishing.
```
Comment (BigRoy) [inline website/docs/addon_maya_ornatrix_artist.md:93]:
```suggestion
Create an ```Ornatrix Cache``` instance or load an ornatrix rig which automatically creates the instance ready for publishing.
```

---

Question: The ornatrix cache and rigs only ever support ONE mesh with ONE ornatrix node, right? Or can it export multiples into one ornatrix product?

And... can you add a paragraph here @moonyuet which describes what you need to put INTO the Ornatrix Cache instance. Is it the mesh? Is it the Ornatrix node? Is it just one mesh or multiple?

File: website/docs/addon_maya_ornatrix_artist.md [hunk 16]
```diff
@@ -0,0 +1,100 @@
...
+- Ornatrix Almebic (.abc)
+- Cache Setting data (.cachesettings)
+
+### Loading
+
+![Maya - Ornatrix Loader Product Type](assets/maya/artist/ox_loader_product_type.png)
+
+**Ornatrix Rig** supports to load maya scene (.ma) and Ornatrix alembic (.abc).
+
+![Maya - Ornatrix Loaders](assets/maya/artist/ox_rig_loader.png)
```
Comment (BigRoy) [inline website/docs/addon_maya_ornatrix_artist.md:64]: Can we update this screenshot so the Hair to Guide loader isn't there?

**Review by BigRoy — APPROVED** [review]
Great work!

**Pull request merged**